src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -22,3 +22,27 @@
                          [selection]="expandedRow">
   </cd-rgw-bucket-details>
 </cd-table>
+
+<ng-template #bucketSizeTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_size > 0); else noSizeQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.size_usage">
+  </cd-usage-bar>
+
+  <ng-template #noSizeQuota>
+  {{row.size_usage | dimlessBinary}}
+  </ng-template>
+</ng-template>
+
+<ng-template #bucketObjectTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_objects > 0); else noObjectQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.object_usage">
+  </cd-usage-bar>
+
+  <ng-template #noObjectQuota>
+  {{row.object_usage | dimlessBinary}}
```

Comment:
You should use ```dimless``` (not ```dimlessBinary```) as suggested by @Devp00l here:
https://github.com/ceph/ceph/pull/34606#discussion_r429959080

Reply:
I guess that's for the 'no of objects' only , as other data are in bytes so have to use dimlessBinary

Reply:
@alfonsomthd the template name is explained [here](https://github.com/ceph/ceph/pull/34606#discussion_r430024115)

Reply:
As mentioned in IRC:
Why is the pipe in use here? `object_usage` resamples a percentage number (from 0 to 1). May the wrong attribute is piped here?

Reply:
Yes that's need to be fixed, I initially confused it with the size

Reply:
But we have to use dimlessPipe instead as 'noObjectQutoa' represents the case when there is no quota limit so the object usage = number of objects (and they can be in large numbers)

Reply:
That's really not so obvious. How about
```suggestion
  {{row.num_objects | dimlessBinary}}
```
As the template is only used if that's the case than we can just use it instantly and not mix the scope of the variable.

Reply:
@Devp00l @avanthakkar @epuertat 
In my environment I set 1MB for size & 10 objects quota:
```radosgw-admin quota set --quota-scope=bucket --uid=dev --bucket test --max-objects=10 --max-size=1M
```
The size tooltip data is not meaningful:
![size-tooltip-bad](https://user-images.githubusercontent.com/6780162/83250071-d433db00-a1a7-11ea-9087-c6d728b05934.png)

After changing the code I see the right data:
```
  <cd-usage-bar *ngIf="(row.bucket_quota.max_size > 0); else noSizeQuota"
             [totalBytes]="row.bucket_quota.max_size"
             [usedBytes]="row.bucket_size">
```
![size-tooltip-good](https://user-images.githubusercontent.com/6780162/83249550-06910880-a1a7-11ea-99ba-0017359f88c2.png)

Current object tooltip:
![object-tooltip-bad](https://user-images.githubusercontent.com/6780162/83249589-16105180-a1a7-11ea-8055-d5f2df5c5a70.png)

After changing the code I see the right data BUT <cd-usage-bar> assumes bytes instead of
raw integers:
```
  <cd-usage-bar *ngIf="(row.bucket_quota.max_objects > 0); else noObjectQuota"
             [totalBytes]="row.bucket_quota.max_objects"
             [usedBytes]="row.num_objects">
```
![object-tooltip-good-almost](https://user-images.githubusercontent.com/6780162/83249739-540d7580-a1a7-11ea-94b8-51a60829d3db.png)

So for object usage bar I see 2 options:
1) Extend <cd-usage-bar> functionality in order to deal with raw integers also
2) Replace with another usage bar that fits here

Thoughts?

Reply:
Thanks @alfonsomthd, hadn't that on my plate. If this is the case we should extend `cd-usage-bar` to be able to use a given pipe instead `dimlessBinary` or just use a not binary flag to use `dimless` instead - as no other cases for other pipes come to my mind this should be just fine.

Reply:
I think extending functionality for raw integers can be a better option